packages/odyssey-react-mui/src/EmptyState.tsx

```diff
@@ -28,37 +28,51 @@ const EmptyContainer = styled("div", {
   padding: odysseyDesignTokens.Spacing5,
   textAlign: "center",
   width: "100%",
+  alignItems: "center",
 }));
 
-export type DataTableEmptyStateProps = {
+export type EmptyStateProps = {
+  /**
+   * Main heading of the empty state
+   */
   heading: string;
+  /**
+   * A descriptive text explaining more context as to why we don't have data.
+   */
   text: string;
```

Comment:
I think this should be named `description` right? `text` is a bit generic. The intent is for it to describe the heading similar to an `aria-description`.